app/javascript/packages/verify-flow/steps/password-confirm/password-confirm-step.tsx

```diff
@@ -1,13 +1,33 @@
 import type { ChangeEvent } from 'react';
+import { useContext } from 'react';
+import { t } from '@18f/identity-i18n';
+import { Button } from '@18f/identity-components';
+import { FormStepsButton, useHistoryParam } from '@18f/identity-form-steps';
 import { PasswordToggle } from '@18f/identity-password-toggle';
-import { FormStepsButton } from '@18f/identity-form-steps';
 import { Alert } from '@18f/identity-components';
 import type { FormStepComponentProps } from '@18f/identity-form-steps';
+import { ForgotPassword } from './forgot-password';
+import { VerifyFlowContext } from '@18f//identity-verify-flow';
 import type { VerifyFlowValues } from '../../verify-flow';
 
 interface PasswordConfirmStepStepProps extends FormStepComponentProps<VerifyFlowValues> {}
 
 function PasswordConfirmStep({ errors, registerField, onChange }: PasswordConfirmStepStepProps) {
+  const { basePath } = useContext(VerifyFlowContext);
+  const [path, setPath] = useHistoryParam(basePath);
+
+  function goToForgotPassword() {
+    setPath('forgot_password');
+  }
+
+  function goBack() {
+    setPath('password_confirm');
+  }
+
+  if (path === 'forgot_password') {
+    return <ForgotPassword goBack={goBack} />;
+  }
```

Comment:
It occurs to me now one more thing we'll need to make sure here is that we don't trigger an accessibility issue with focus loss through navigation ([related YouTube video](https://www.youtube.com/watch?v=srLRSQg6Jgg)). `FormSteps` has a `onPageTransition` helper function here to assist with this, which we can add as a property to props, which will automatically focus the first element of the new content. One challenge we'll need to sort through is when to call it -- I expect it might be best to do so as a `useEffect` of the changed path.

Maybe something like...

```tsx
import { useDidUpdateEffect } from '@18f/identity-react-hooks';

function PasswordConfirmStep({ onPageTransition }: PasswordConfirmStepStepProps) {
  useDidUpdateEffect(onPageTransition, [path]);
```